sources/3.0.0/sections/12-metadata.adoc

```diff
@@ -481,21 +481,20 @@ CI_Responsibility>CI_Organisation
 
 |Attribute
 |replacedData
-//Earlier comment from PT11 in change guide states: "Add, 'Not used in S-102'. For service management, RENCs and service providers will need to use this attribute to manage coverage changes."
-//If not used, this row should be deleted and the remark in 'Classname' row updated. Does the PT11 comment mean it is needed after all? (RM Metacomment 24Jan2023)
-|If a data file is cancelled, it is replaced by another data file.
+|Indicates if a cancelled dataset is replaced by another data file(s)
 |0..1
 |Boolean
-|
+|See <<subsec-s102-replacedData-dataReplacement-note1>> +
+*Mandatory when purpose = cancellation*
 
 |Attribute
 |dataReplacement
-//Earlier comment from PT11 in change guide states: "Add, 'Not used in S-102'. For service management, RENCs and service providers will need to use this attribute to manage coverage changes."
-//If not used, this row should be deleted and the remark in 'Classname' row updated. Does the PT11 comment mean it is needed after all? (RM Metacomment 24Jan2023)
-|Cell name
+|Dataset name
 |0..*
 |CharacterString
-|A dataset may be replaced by 1 or more datasets.
+|A dataset may be replaced by 1 or more datasets. +
```

Comment:
In my opinion, an example makes sense here.
Is the name
102DE00XXXXX.H5
or
102DE00XXXXX
?